packages/lib/models/Note.ts

```diff
@@ -192,6 +192,11 @@ export default class Note extends BaseItem {
 			pathsToTry.push(Resource.baseRelativeDirectoryPath());
 		}
 
+		body = Note.replacePaths(pathsToTry, body);
+		return body;
+	}
+
+	static replacePaths(pathsToTry: string[], body: string) {
```

Comment:
Please don't create a separate function, that's not necessary here

Reply:
It is hard to write unit tests for `replaceResourceExternalToInternalLinks` because the function calls other functions which output could change depending on the system (e.g. calling `Setting.value` or `shim.pathRelativeToCwd`). I created a new function such that it does not call such functions. It make it easy to write unit tests.
Another way to write unit tests is to mock the output of  `Setting.value` and `shim.pathRelativeToCwd` such that they return a fixed value. This will work, but the tests may break if we change the implementation of the function.
I prefer the first option, but I can implement the second option or even another if you have a better idea.

Reply:
Name it "replaceResourceExternalToInternalLinks_", make it private, and add a comment: "This is a moved to a separate function for the purpose of testing only".

Reply:
Done